src/pymedphys/dicom/_level3/dicom_collection.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (C) 2019 Simon Biggs, Matt Jennings
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version (the "AGPL-3.0+").
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License and the additional terms for more
+# details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+
+from copy import deepcopy
+
+import numpy as np
+import pydicom
+
+from .._level1.create import dicom_dataset_from_dict
+from .._level2.anonymise import anonymise_dicom_dataset
+from .._level2.dose import (
+    extract_dicom_patient_xyz, convert_xyz_to_dicom_coords)
+
+from ...libutils import get_imports
+
+IMPORTS = get_imports(globals())
+
+# pylint: disable=W0201
+
+
+# Many thanks to the Medical Connections for offering free
+# valid UIDs (http://www.medicalconnections.co.uk/FreeUID.html)
+# Their service was used to obtain the following root UID for PyMedPhys:
+PYMEDPHYS_ROOT_UID = '1.2.826.0.1.3680043.10.188'
+
+
+class DicomBase:
+    def __init__(self, dataset, copy=True):
+        if copy:
+            dataset = deepcopy(dataset)
+
+        if dataset.is_little_endian is None:
+            dataset.is_little_endian = True
+
+        if dataset.is_implicit_VR is None:
+            dataset.is_implicit_VR = True
+
+        self.dataset = dataset
+
+    @classmethod
+    def from_file(cls, filepath):
+        dataset = pydicom.read_file(filepath, force=True)
```

Comment:
I think `pydicom.dcmread(filepath)` is the standard file reading function. See https://pydicom.github.io/pydicom/stable/ref_guide.html

Is there a reason you've opted for `pydicom.read_file(filepath)`?

Reply:
Nope, no good reason. I'll swap over.